src/wp-includes/ms-functions.php

```diff
@@ -954,7 +954,7 @@ function wpmu_signup_blog_notification( $domain, $path, $title, $user_login, $us
 	$message_headers = "From: \"{$from_name}\" <{$admin_email}>\n" . 'Content-Type: text/plain; charset="' . get_option( 'blog_charset' ) . "\"\n";
 
 	$user            = get_user_by( 'login', $user_login );
-	$switched_locale = switch_to_locale( get_user_locale( $user ) );
+	$switched_locale = switch_to_user_locale( $user->ID );
```

Comment:
If `get_user_by()` returns `false`, `$user->ID` will throw a warning `Attempt to read property "ID" on bool`.

Previously, `switch_to_locale()` handled a `false` value by returning `false`.

I suggest a change to:

```suggestion
	$switched_locale = $user instanceof WP_User ? switch_to_user_locale( $user->ID ) : false;
```